engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/expression/builtin/meta/TypeOfNodeMultiValueTest.java

```diff
@@ -0,0 +1,150 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.oracle.truffle.api.interop.TruffleObject;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.enso.interpreter.runtime.data.EnsoMultiValue;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.error.DataflowError;
+import org.enso.interpreter.runtime.library.dispatch.TypeOfNode;
+import org.enso.interpreter.test.ValuesGenerator;
+import org.enso.test.utils.ContextUtils;
+import org.enso.test.utils.TestRootNode;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Value;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class TypeOfNodeMultiValueTest {
+  @Parameterized.Parameter(0)
+  public Object value;
+
+  @Parameterized.Parameter(1)
+  public String type;
+
+  @Parameterized.Parameter(2)
+  public int typeIndex;
+
+  private static Context ctx;
+
+  private static Context ctx() {
```

Comment:
```suggestion
      ctx = ContextUtils.createDefaultContext();
```

Reply:
This might make a difference in initialization time for example in `testOnly` command - e.g. when the class is loaded, but not test is executed, right?